packages/block-library/src/post-title/block.json

```diff
@@ -5,7 +5,26 @@
 		"postId",
 		"postType"
 	],
+	"attributes": {
+		"align": {
+			"type": "string"
+		},
+		"level": {
+			"type": "number",
+			"default": 2
+		}
+	},
 	"supports": {
-		"html": false
+		"html": false,
+		"lightBlockWrapper": true,
+		"__experimentalSelector": {
+			"core/post-title/h1": "h1",
+			"core/post-title/h2": "h2",
+			"core/post-title/h3": "h3",
+			"core/post-title/h4": "h4",
+			"core/post-title/h5": "h5",
+			"core/post-title/h6": "h6",
+			"core/post-title/p": "p"
```

Comment:
:wave: @apeatling I'm trying to understand why do we need the `core/post-title/p` here? It looks like this block behaves like the heading block: it can have 1-6 levels, but that's about it, right?

Reply:
Prepared https://github.com/WordPress/gutenberg/pull/24418 to fix it.

Reply:
I loosely recall there being support for using P as a selector in a previous version of the block, that I removed. This was left over. I might have that wrong but it was something like that. In any case, removing it makes sense.